### PR TITLE
Gradients train

### LINKED ISSUE #174: Add support for linear-gradient in ron
Please add support for "linear-gradient" in css files.

### PR DESCRIPTION
**Pull request by sandmor**
# Context:
This PR fix #174, adding support for linear-gradients.

## Contribution checklist:
- [ ] Complete the angle to ends convertion algorithm
- [ ] Add [documentation](https://doc.rust-lang.org/1.7.0/book/documentation.html) to all public structs, traits and functions.
- [ ] Add unit tests if possible
- [ ] Describe the major change(s) in the CHANGELOG.MD

### CODE REVIEW

File: crates/utils/src/angle.rs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+use derive_more::{Add, Div, From, Mul, Sub};
+use std::f64::consts::PI;
+
+/// The OrbTk way to manage angles
+#[derive(Add, Sub, Copy, From, Clone, Debug, PartialEq, Mul, Div)]
```
Comment (FloVanGH) [inline crates/utils/src/angle.rs:5]: Please add documentation to all public methdos.

File: crates/utils/src/expression.rs [hunk 2]
```diff
@@ -76,10 +76,140 @@ impl Expression {
         }
     }
 
+    pub fn gradient_stop(&self) -> Option<GradientStop> {
```
Comment (FloVanGH) [inline crates/utils/src/expression.rs:79]: Please add documentation to public methods.

File: crates/theme/assets/dark/dark.ron [hunk 3]
```diff
@@ -33,7 +33,8 @@ Theme (
                 "height": 36,
                 "foreground": "$LINK_WATER",
                 "icon_brush": "$LINK_WATER",
-                "background": "$LYNCH",
+                //"background": "$LYNCH",
+                "background": "linear-gradient(red, blue)",
```
Comment (FloVanGH) [inline crates/theme/assets/dark/dark.ron:37]: This was for testing purpose? Can you undo this please?

File: examples/minimal.rs [hunk 4]
```diff
@@ -10,7 +10,7 @@ fn main() {
                 .title("OrbTk - minimal example")
                 .position((100.0, 100.0))
                 .size(420.0, 730.0)
-                .child(TextBlock::new().text("OrbTk").margin(4.0).build(ctx))
+                .child(Button::new().text("OrbTk").margin(4.0).build(ctx))
```
Comment (FloVanGH) [inline examples/minimal.rs:13]: This was for testing purpose? Can you undo this please?

File: crates/theme/assets/dark/dark.ron (hunk 3 above)
Comment (sandmor) [reply crates/theme/assets/dark/dark.ron:37]: Yeah, I was sure that I remove it

**Pull request closed without merging**